README.md

```diff
@@ -0,0 +1,40 @@
+
+<section>
+<div align="center">
+    <a href="https://github.com/jonathanjonathan001/springbootlab/issues">
+        <img src="https://img.shields.io/github/issues-raw/jonathanjonathan001/springbootlab" alt="Open issues workflow badge"/>
+    </a>
+    <a href="https://github.com/jonathanjonathan001/springbootlab/pulls">
+        <img src="https://img.shields.io/github/issues-pr/jonathanjonathan001/springbootlab" alt="Pull request workflow badge"/>
+    </a>
+    <a href="https://github.com/jonathanjonathan001/springbootlab/issues?q=is%3Aissue+is%3Aclosed">
+        <img src="https://img.shields.io/github/issues-closed-raw/jonathanjonathan001/springbootlab" alt="Open issues workflow badge"/>
```

Comment:
nit-picking here :) , but on line 11, alt-text should say Closed issues workflow badge, right?

Reply:
Oh, thats right. Fixed it now